UserTools/Unity.h

```diff
@@ -112,10 +112,9 @@
 #include "ClusterFinder.h"
 #include "RunValidation.h"
 #include "AmBeRunStatistics.h"
-#include "ClusterFinder.h"
```

Comment:
did you mean to delete these two tools?

Reply:
I only wanted to delete the ClusterFinder, since it was twice in the Unity file. I didn't mean to delete the ClusterClassifiers tool, sorry about that.